admin/views/tabs/dashboard/features.php

```diff
@@ -57,16 +58,31 @@
 			$name .= ' ' . new Premium_Badge_Presenter( $feature->name );
 		}
 
+		if ( ! empty( $feature->in_beta ) && $feature->in_beta === true ) {
+			$name .= ' ' . new Beta_Badge_Presenter( $feature->name );
+		}
```

Comment:
Does this have to be added on the multisite feature toggle page as well? We suspect so! :-)